02-dz0/01-semantic-mockup/style.css

```
@@ -0,0 +1,84 @@
+*
+{
```

Обычно используется такое форматирование:
```suggestion
.any_selector {
    font-family: Arial, Helvetica, sans-serif;
}
```

---

Исправил